### ISSUE TITLE
Import objects as tscns?

### ISSUE THREAD

**Issue by elvisish** [vnen/godot-tiled-importer#169]
Is possible to import objects in TilEd as tscn scenes?

**Comment by cromerc** [comment]
That's what this plugin does. It imports them as scenes that can be used in Godot. Literally the first feature listed in the README when you open this repo...

**Comment by elvisish** [comment]
> That's what this plugin does. It imports them as scenes that can be used in Godot. Literally the first feature listed in the README when you open this repo...

No, read what I said, the objects not the tilemaps, like things used for player or enemy placements, that isn't obvious at all in the documentation.

**Comment by cromerc** [comment]
Sorry misread that. In that case, no this plugin does not do that. They are not separate scenes, they are nodes of specific types depending on the type of object in tiled.

**Comment by erniel** [comment]
not yet possible but you can do it with post import script.
